commands/project/migrateApp.ts

```diff
@@ -1,283 +1,86 @@
-  logger.log('');
-  logger.log(i18n(`${i18nKey}.warning.existingApps`));
-  logger.log('');
-  logger.log(i18n(`${i18nKey}.warning.copyApp`));
-  uiLine();
-
-  const { shouldCreateApp } = await promptUser({
-    name: 'shouldCreateApp',
-    type: 'confirm',
-    message: i18n(`${i18nKey}.createAppPrompt`),
-  });
-  process.stdin.resume();
-
-  if (!shouldCreateApp) {
-    process.exit(EXIT_CODES.SUCCESS);
-  }
-
-  try {
-    SpinniesManager.init();
-
-    SpinniesManager.add('migrateApp', {
-      text: i18n(`${i18nKey}.migrationStatus.inProgress`),
-    });
-
-    handleKeypress(async key => {
-      if ((key.ctrl && key.name === 'c') || key.name === 'q') {
-        SpinniesManager.remove('migrateApp');
-        logger.log(i18n(`${i18nKey}.migrationInterrupted`));
-        process.exit(EXIT_CODES.SUCCESS);
-      }
-    });
-
-    const { data: migrateResponse } = await migrateApp(
-      derivedAccountId,
-      appId,
-      projectName
-    );
-    const { id } = migrateResponse;
-    const pollResponse = await poll(() =>
-      checkMigrationStatus(derivedAccountId, id)
-    );
-    const { status, project } = pollResponse;
-    if (status === 'SUCCESS') {
-      const absoluteDestPath = path.resolve(getCwd(), projectDest);
-      const { env } = accountConfig;
-      const baseUrl = getHubSpotWebsiteOrigin(env);
-
-      const { data: zippedProject } = await downloadProject(
-        derivedAccountId,
-        projectName,
-        1
-      );
-
-      await extractZipArchive(
-        zippedProject,
-        sanitizeFileName(projectName),
-        path.resolve(absoluteDestPath),
-        { includesRootDir: true, hideLogs: true }
-      );
-
-      SpinniesManager.succeed('migrateApp', {
-        text: i18n(`${i18nKey}.migrationStatus.done`),
-        succeedColor: 'white',
-      });
-      logger.log('');
-      uiLine();
-      logger.success(i18n(`${i18nKey}.migrationStatus.success`));
-      logger.log('');
-      logger.log(
-        uiLink(
-          i18n(`${i18nKey}.projectDetailsLink`),
-          `${baseUrl}/developer-projects/${derivedAccountId}/project/${encodeURIComponent(
-            project.name
-          )}`
-        )
-      );
-      process.exit(EXIT_CODES.SUCCESS);
-    }
-  } catch (error) {
-    await trackCommandMetadataUsage(
-      'migrate-app',
-      { status: 'FAILURE' },
-      derivedAccountId
-    );
-    SpinniesManager.fail('migrateApp', {
-      text: i18n(`${i18nKey}.migrationStatus.failure`),
-      failColor: 'white',
-    });
-    if (error.errors) {
-      error.errors.forEach(logError);
-    } else {
-      logError(error, new ApiErrorContext({ accountId: derivedAccountId }));
-    }
-
-    process.exit(EXIT_CODES.ERROR);
-  }
-  await trackCommandMetadataUsage(
-    'migrate-app',
-    { status: 'SUCCESS' },
-    derivedAccountId
-  );
-  process.exit(EXIT_CODES.SUCCESS);
-};
+export function builder(yargs: Argv): Argv<MigrateAppOptions> {
```

Comment:
Will it confuse folks to see these options listed in the help menu when this is technically the deprecated command?

Reply:
The only new option added is the `--platform-version` and it is hidden, so I think we are good.